saleor/checkout/utils.py

```diff
@@ -880,8 +880,8 @@ def create_order(cart: Cart, tracking_code: str, discounts, taxes):
     return order
 
 
-def is_fully_paid(cart: Cart, taxes, discounts):
-    """Check if checkout is fully paid."""
+def can_be_fully_paid(cart: Cart, taxes, discounts):
```

Comment:
I think the original name was fine, I don't see a reason why we should change it.

Reply:
Because this method only checks ```payment.total``` instead of ```payement.captured``` which means the order may not be charged at all but only have ability to be charged later. With this reason , I think ```**can_be**_fully_paid``` is more approprate than ```**is**_fully_paid```, what do you think?

Reply:
Ok, I got what you mean. We don't check `captured` amount at this point on purpose - at the point when this method is run we only require the customer to have provided a valid payment method for the required amount.